aptos-move/aptos-vm/src/data_cache.rs

```diff
@@ -5,27 +5,26 @@
 
 use crate::{
     aptos_vm_impl::gas_config,
-    move_vm_ext::{get_max_binary_format_version, AptosMoveResolver},
+    move_vm_ext::{get_max_binary_format_version, AptosMoveResolver, AsExecutorView},
+    storage_adapter::ExecutorViewBase,
 };
 #[allow(unused_imports)]
 use anyhow::Error;
-use aptos_aggregator::{
-    aggregator_extension::AggregatorID,
-    resolver::{AggregatorReadMode, AggregatorResolver},
-};
-use aptos_framework::natives::state_storage::StateStorageUsageResolver;
-use aptos_state_view::{StateView, TStateView};
+use aptos_aggregator::resolver::TAggregatorView;
+use aptos_state_view::{StateView, StateViewId};
```

Comment:
Based on the above changes, I thought StateView is replaced by ExecutorView. We still have StateView? What's the difference between StateView and ExecutorView?

Reply:
StateView is a storage concept that was abused, ExecutorView wraps the resolvers that are needed to get things from Executor (which to the Move VM is like storage, but not really)